notebooks/workflow_cost_estimator/main.py

```diff
@@ -48,9 +48,12 @@ def list_submissions_chronological():
     def cost_for_submission(submission_id: str):
         submission = workflows.get_submission(submission_id)
         for wf in submission['workflows']:
-            for shard_info in workflows.estimate_workflow_cost(submission_id, wf['workflowId']):
-                shard_info['workflow_id'] = wf['workflowId']
-                yield shard_info
+            try:
+                for shard_info in workflows.estimate_workflow_cost(submission_id, wf['workflowId']):
+                    shard_info['workflow_id'] = wf['workflowId']
+                    yield shard_info
+            except KeyError:
+                print("No workflow ID found. Usually this means a workflow didn't even start due to Terra detecting an invalid input.")
```

Comment:
Let's move speculation of cause to the markdown at the top of the notebook. e.g. "Submissions that are in progress may contain incomplete or pending jobs, reducing cost estimate accuracy. Pending jobs may not be included in the report."

Reply:
In my new commit, I changed the output of one of the middle cells to help users figure out why they can't get an estimate for a given workflow, and felt that information should be close to the cell that gives them that output instead of at the top. Let me know if you think it's still better at the top.

Reply:
I’d prefer all the caveats at the top. Perhaps an enumeration of reasons users may experience difficulty with the notebook:
  - you fool, you ran it too soon!
  - After 40 days, no bueno
  - etc.

Also, it’d be preferable to use either the logging module, or the warnings module, to output warnings. These are handled differently than stdout in Jupyter notebooks.